hoodie-common/src/main/java/com/uber/hoodie/common/table/log/block/HoodieAvroDataBlock.java

```diff
@@ -85,7 +91,9 @@ public Schema getSchema() {
     output.writeInt(records.size());
 
     // 4. Write the records
-    records.forEach(s -> {
+    Iterator<IndexedRecord> itr = records.iterator();
+    while(itr.hasNext()) {
```

Comment:
same here.. please check formatting in terms of space between keyword and `(` everywhere ..